### DISCUSSION TITLE
[Question] Navigate to the message from search

### DISCUSSION THREAD

**Discussion by jasnicaa** [GetStream/stream-chat-flutter#685] [Q&A] [answered]
Hey guys, this is more of a question because I am unable to find it in the documentation. 

So I want to search through the channel and be able to navigate to the specific message I was searching for. What is the best way to accomplish that? I haven't done this before in Flutter, in iOS we had scrollToRowAtIndexPath and I guess there is something the same in Android, but for the Flutter I saw that there isn't some straightforward solution. 

I was digging through your implementation for navigation to the thread messages and was wondering should I take a similar approach?

Thanks.

**Comment by imtoori** [comment, marked as answer]
hey @jasnicaa take a look at this implementation in our sample app, I think it will help you a lot 
https://github.com/GetStream/flutter-samples/blob/main/packages/stream_chat_v1/lib/channel_list.dart#L82

**Reply by jasnicaa** [reply]
aaa yes thanks! I should've looked better
